python/mxnet/_numpy_op_doc.py

```diff
@@ -20,6 +20,107 @@
 """Doc placeholder for numpy ops with prefix _np."""
 
 
+def _np_all(a, axis=None, keepdims=False, out=None):
```

Comment:
Order of params is still different, you'll probably need to add the python wrapper so that the order of arguments is consistent with official NumPy. Same for `any`